tests/test_modifying_name_and_symbol.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from brownie import Contract, Wei, reverts
+from fixedint import *
+import shared
+
+def test_modifying_name_and_symbol(accounts, loanToken, loanTokenLogic, name, symbol, LoanToken, LoanTokenLogicStandard):
+    
+    localLoanToken = Contract.from_abi("loanToken", address=loanToken.address, abi=LoanToken.abi, owner=accounts[0])
+    localLoanToken.setTarget(loanTokenLogic.address)
+    localLoanToken = Contract.from_abi("loanToken", address=loanToken.address, abi=LoanTokenLogicStandard.abi, owner=accounts[0])
```

Comment:
This is not necessary, because we don't have different targets anymore (token logic is the target by default).

Delete the lines  8 + 9

Do the same for all of the other test cases you modified. No more settings targets, just load the loanToken directly with the logic ABI.